flink-streaming-java/src/main/java/org/apache/flink/streaming/api/graph/StreamingJobGraphGenerator.java

```diff
@@ -130,7 +130,7 @@ private JobGraph createJobGraph() {
 		jobGraph.setScheduleMode(ScheduleMode.EAGER);
 
 		// Generate deterministic hashes for the nodes in order to identify them across
-		// submission iff they didn't change.
+		// submission if they didn't change.
```

Comment:
I think here means `if and only if` and is not a typo.

Reply:
There are indeed abbreviations of this form, and there is no problem with this understanding. Agree.

Reply:
I am reverting this change.